site/gatsby-site/src/pages/apps/csettool/{mongodbAiidprodIncidents.incident_id}.js

```diff
@@ -16,7 +16,7 @@ const ToolPage = (props) => {
 
   const { data, loading } = useQuery(FIND_CLASSIFICATION, {
     variables: {
-      query: { incidents_in: { incident_id: parseInt(incident_id) }, namespace_in: allNamespaces },
+      filter: { incidents: { IN: parseInt(incident_id) }, namespace: { IN: allNamespaces } },
```

Comment:
Any other places we have to `parseInt`? I've been looking as I go...

Reply:
Not that I aware.........